SPHINXsys/cmake/Dirsearch_shared.cmake

```diff
@@ -31,7 +31,7 @@ MACRO(SOURCE_DIRECTORIES_SHARED return_list)
         if(BUILD_WITH_SIMBODY)
             SET(dir_list ${dir_list} ${dir_path})
         else() # if Simbody is not built, ignore the simbody and clapack folders
-            if(NOT dir_list MATCHES ("/simbody/+" OR "/clapack_for_SPHinXsys/+") )
+            if(NOT (dir_list MATCHES "/simbody/+" OR  dir_list MATCHES "/clapack_for_SPHinXsys/+") )
```

Comment:
You should remove this clapack_for_SPHinXsys, it doesn't exist anymore. We don't build clapack at all, it's handled in simbody. The necessary lapack function is replaced by eigen.

Reply:
Done, thanks.